App.jsx

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react'
```

Comment:
Remove this import. It is not getting used.

Reply:
done... Kindly review PR